airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_spec.py

```diff
@@ -682,6 +682,230 @@ def test_enum_usage(connector_spec, should_fail):
             ),
             "",
         ),
+        # SUCCESS: Fields specified in advanced_auth exist in spec, Oauth is default
```

Comment:
Should we have a test where `predicate_key.oneOf` does not exist in which case, the test should pass?

Reply:
`predicate_key.oneOf`does not exist  - you mean "credentials" don't have "oneOf" field? This test case is already added https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_spec.py#L628 . The test passes due to  only one option to auth so we don't need to check default.